### PR TITLE
Trim Quotes (single and double) from $file.

### PR DESCRIPTION
**Pull request by kopepasah**
See #927.

### CODE REVIEW

File: WordPress/Sniffs/Files/FileNameSniff.php [hunk 1]
```diff
@@ -127,7 +127,7 @@ public function register() {
 	 */
 	public function process_token( $stackPtr ) {
 
-		$file     = $this->phpcsFile->getFileName();
+		$file     = trim( $this->phpcsFile->getFileName(), "\x22\x27" );
```
Comment (jrfnl) [inline WordPress/Sniffs/Files/FileNameSniff.php]: WPCS actually contains a utility function to do quote stripping properly & consistently, you can call it using `$this->strip_quotes( $file )`.

https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/blob/develop/WordPress/Sniff.php#L865-L867

**Comment by kopepasah** [comment]
@jrfnl updated to use the `strip_quotes` method and added an inline comment.

**Comment by jrfnl** [comment]
@kopepasah Thanks for the update. Looks good to me.

**Comment by JDGrimes** [comment]
Thanks @kopepasah !

**Pull request merged**